src/js/webrequest.js

```diff
@@ -701,7 +701,14 @@ function dispatcher(request, sender, sendResponse) {
     let tab_id = request.tabId,
       tab_url = request.tabUrl,
       tab_host = window.extractHostFromURL(tab_url),
-      has_tab_data = badger.tabData.hasOwnProperty(tab_id);
+      has_tab_data = badger.tabData.hasOwnProperty(tab_id),
+      errorText = null;
```

Comment:
Nit: Non-object variables should use underscores: `error_text`.

Reply:
Done.